test/build/dotenv-webpack-plugin/builtin-config/.env

```diff
@@ -0,0 +1,2 @@
+WEBPACK_VARIABLE1=value1
+WEBPACK_VARIABLE2=value2
```

Comment:
Let's also add tests with `.env.local` and `.env.[mode].local`

Reply:
added